linux/LinuxProcessList.c

```diff
@@ -766,7 +766,7 @@ static int handleNetlinkMsg(struct nl_msg *nlmsg, void *linuxProcess) {
    struct nlmsghdr *nlhdr;
    struct nlattr *nlattrs[TASKSTATS_TYPE_MAX + 1];
    struct nlattr *nlattr;
-   struct taskstats *stats;
```

Comment:
Please drop the pointer variable `stats` and replace its usages with `&stats_data`

Reply:
It was easier to just make `stats` a stack variable and change `->` to `.`